Classes/Repository/RepositoryViewController.swift

```diff
@@ -12,18 +12,27 @@ import Pageboy
 import TUSafariActivity
 import SafariServices
 
+enum RepositoryPageIndex: Int {
+    case overview    = 0
+    case issues      = 1
+    case pullRequest = 2
+    case code        = 3
```

Comment:
What's to stop this from getting out of sync w/ the actual pages? This will be really easy to break and the compiler wont help us.